v3/src/components/codap-component.scss

```diff
@@ -52,6 +53,7 @@ $corner-drag-size: calc($border-drag-width * 2);
   position: absolute;
   margin: 0;
   user-select: none;
+  z-index: 1000;
```

Comment:
I assume this is more `z-index` machinations to work around leaflet. We should set up a [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context) so that leaflet's `z-index` values don't affect the surrounding content. The simplest map-specific solution would seem to be adding a `z-index` value to the `.map-container` `div`, given that one of the situations that creates a new stacking context is "Element with a [`position`](https://developer.mozilla.org/en-US/docs/Web/CSS/position) value `absolute` or `relative` and [`z-index`](https://developer.mozilla.org/en-US/docs/Web/CSS/z-index) value other than auto." Arguably, we could/should also add a `z-index` to the `.codap-component` `div` to create a stacking context for the entire tile, but that probably warrants a larger conversation.